client/src/containers/ResetPasswordContainer.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import React from 'react';
+import ResetPassword from '../components/ResetPassword';
+import { resetPasswordFromToken } from '../actions/userActions';
+
+const ResetPasswordContainer = (props) => {
+  function onResetPassword(formData) {
+    props.resetPasswordFromToken(formData.password, props.location.query.id, props.location.query.token);
```

Comment:
Our linting requires these props being defined in propTypes on the component (see: https://facebook.github.io/react/docs/typechecking-with-proptypes.html)